libsovtoken/src/logic/parsers/parse_get_txn_fees.rs

```diff
@@ -79,7 +79,7 @@ pub fn get_fees_state_proof_extractor(reply_from_node: *const c_char, parsed_sp:
 
     // TODO: Make sure JSON serialisation preserves order
     let kvs_to_verify = KeyValuesInSP::Simple(KeyValueSimpleData {
-        kvs: vec![(base64::encode(FEES), Some(fees.to_string()))],
+        kvs: vec![(base64::encode(FEES_STATE_KEY), Some(fees.to_string()))],
```

Comment:
What about response for GET_FEE txn where the key is `200:<fee_alias>`?

Reply:
there is no such functionality in libsovtoken and libindy payment API now